modules/tinymce/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `editor.selection.getContent()` API did not respect the `no_events` argument #TINY-8018
 - The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API #TINY-8018
 - The `table` plugin would sometimes not correctly handle headers in the `tfoot` section #TINY-8104
+- Deleting content would sometimes not fire `beforeinput` and `input` events as expected #TINY-8168
```

Comment:
```suggestion
- Deleting content would sometimes not fire `input` events as expected #TINY-8168
```
We moved the `beforeinput` logic to a follow up Jira

Reply:
Right I thought that had been updated as well I should have double checked that.